validation_tests/vtk-m/run.sh

```diff
@@ -2,6 +2,4 @@
 
 source setup.sh
 
-set -x
-
-./build/smoke_test
+spackTestRun $VTK-M_HASH
```

Comment:
I'm not sure where VTK-M_HASH comes from for you but I don't think spack populates it when I load vtk-m and bash does not handle that string. 
```
==> Error: Unexpected token: '-'  Encountered when parsing spec:
    /-M_HASH
```

Reply:
apparently `setup.sh` does a tr '-' '_' for the PKG_HASH variable. I corrected the variable name in the last push